HollowKnightSplitSettings.cs

```diff
@@ -1358,6 +1364,122 @@ public enum SplitName {
         [Description("Pale Ore (Obtain)"), ToolTip("Splits when obtaining a Pale Ore")]
         OnObtainPaleOre,
 
+        [Description("36000 Geo (Geo Count)"), ToolTip("Splits when geo count ≥36000")]
+        Geo36000,
+
+        [Description("Colo 1 Wave 1a"), ToolTip("Splits upon killing the first Sheilded Fool in wave 1")]

```

Comment:
All of these require a file whose kills* counts have been modified, right? If so, that info should be part of the tooltip for them